.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,37 @@
+name: ⭐️ Feature request
+description: Suggest an idea for this project
+title: "✨ Enhancement: "
+labels: ["✨ Enhancement", "Status: Triage"]
```

Similar here, let's only use `Type: Feature Request`.

```suggestion
labels: ["Type: Feature Request"]
```